es-core/src/components/VideoComponent.cpp

```diff
@@ -221,6 +212,15 @@ void VideoComponent::applyTheme(const std::shared_ptr<ThemeData>& theme, const s
 	if (elem->has("showSnapshotDelay"))
 		mConfig.showSnapshotDelay = elem->get<bool>("showSnapshotDelay");
 
+	setRotation(0.0);
```

Comment:
Theoretical question: should size changed reset the rotation and origin? I know we re-set it afterwards if it exists, but wondering if these two have a purpose?